frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -503,6 +503,10 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
     }
     this.intermissions = newInterms;
     if (!this.initCastsLoaded) {
+      for (let s = 0; s < this.state.step_3.segments.length; s++) {
+        let segment = this.state.step_3.segments[s];
```

Comment:
`const segment` since it's never reassigned.

Reply:
Done.